src/Library/GeneratorService.ts

```diff
@@ -42,20 +52,22 @@ export class GeneratorService {
       .addImport(`../module/${nameFormats.pascalCased}`, importName)
       .modify(/( +)\w+,?\n];\s*$/, { custom: (match: string, space: string) => `${space}${importName},\n${match}` })
       .update();
+
+    output.success(`Created module ${nameFormats.pascalCased}`);
```

Comment:
The service doesn't add output. It's exactly why I moved it to a service. This way a stix-wetland for example can generate a controller, reusing code without polluting the cli with output.

Reply:
So no success message will be displayed or this should be in the GeneratorCommand?